src/browser/StorybookPage.ts

```diff
@@ -96,13 +99,16 @@ function fetchStoriesFromWindow(): Promise<StorybookStory[]> {
 function emitSetCurrentStory(id: string) {
   const storybookPreview = window.__STORYBOOK_PREVIEW__;
 
+  // @ts-expect-error Access the protected "channel", so we can send stuff through it.
+  const channel = storybookPreview.channel;
```

Comment:
`channel` is now "protected" (can only be accessed by subclasses, not externally). Hacking around that here, so we can use it.